Add Advisory 0012 (xz Vulnerability)

bfewster opened the pull request:
Advisory 0012 covers xz vulnerability/backdoor as it relates to Mirantis container products MSR, MCR, and MKE.

Commit:
Add Advisory 0012 (xz Vunerability)

Advisory 0012 covers xz vulerability/backdoor as it relates to Mirantis
contianer products MSR, MCR, and MKE.

ry4nz commented on advisories/0012.md:
MCR was already mentioned earlier at line 10, shall we just use MCR instead of the full name again here?  Same for the MSR and MKE below.

bfewster replied:
I left the paragraphs starting with full name, because it looks awkward starting with an acronym.

ry4nz commented on advisories/0012.md:
```suggestion
### Vulnerability Information
```

Commit:
Resolve Code Review Comments

- Remove duplication of acronym definitions
- Fix heading level of "Vulnerability Information"
